ui/raidboss/data/04-sb/raid/o2n.js

```diff
@@ -0,0 +1,411 @@
+      regexJa: Regexes.losesEffect({ effect: 'レビテト' }),
+      regexCn: Regexes.losesEffect({ effect: '浮空' }),
+      regexKo: Regexes.losesEffect({ effect: '레비테트' }),
+      condition: Conditions.targetIsYou(),
+      run: function(data) {
+        data.levitating = false;
+      },
+    },
+    {
+      id: 'O2N Evilsphere',
+      regex: Regexes.startsUsing({ id: '250F', source: 'Catastrophe' }),
+      regexDe: Regexes.startsUsing({ id: '250F', source: 'Katastroph' }),
+      regexFr: Regexes.startsUsing({ id: '250F', source: 'Catastrophe' }),
+      regexJa: Regexes.startsUsing({ id: '250F', source: 'カタストロフィー' }),
+      regexCn: Regexes.startsUsing({ id: '250F', source: '灾变者' }),
+      regexKo: Regexes.startsUsing({ id: '250F', source: '카타스트로피' }),
+      condition: function(data) {
+        return data.role == 'tank' || data.role == 'healer';
+      },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'O2N -100Gs',
+      regex: Regexes.startsUsing({ id: '24FF', source: 'Catastrophe', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '24FF', source: 'Katastroph', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '24FF', source: 'Catastrophe', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '24FF', source: 'カタストロフィー', capture: false }),
+      regexCn: Regexes.startsUsing({ id: '24FF', source: '灾变者', capture: false }),
+      regexKo: Regexes.startsUsing({ id: '24FF', source: '카타스트로피', capture: false }),
+      infoText: {
+        en: '-100 Gs: Go north/south',
+        de: '-100G: Nach Norden/Süden',
+      },
+      tts: {
+        en: '100 gs',
+        de: '-100 G',
+      },
+    },
+    {
+      id: 'O2N Demon Eye',
+      regex: Regexes.startsUsing({ id: '250D', source: 'Catastrophe', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '250D', source: 'Katastroph', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '250D', source: 'Catastrophe', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '250D', source: 'カタストロフィー', capture: false }),
+      regexCn: Regexes.startsUsing({ id: '250D', source: '灾变者', capture: false }),
+      regexKo: Regexes.startsUsing({ id: '250D', source: '카타스트로피', capture: false }),
+      response: Responses.lookAway(),
+    },
+    {
+      id: 'O2N Earthquake',
+      regex: Regexes.startsUsing({ id: '2512', source: 'Catastrophe', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '2512', source: 'Katastroph', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '2512', source: 'Catastrophe', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '2512', source: 'カタストロフィー', capture: false }),
+      regexCn: Regexes.startsUsing({ id: '2512', source: '灾变者', capture: false }),
+      regexKo: Regexes.startsUsing({ id: '2512', source: '카타스트로피', capture: false }),
+      alertText: function(data) {
+        if (!data.levitating) {
+          return {
+            en: 'Earthquake: Levitate',
```

Comment:
The original o2s triggers were one of the first and doesn't match all the other triggers in some ways.  I'd make this just "levitate" for brevity.